base/math.jl

```diff
@@ -716,7 +716,10 @@ end
 ## rem2pi-related calculations ##
 
 function add22condh(xh::Float64, xl::Float64, yh::Float64, yl::Float64)
-    # as above, but only compute and return high double
+    # This algorithm, due to Dekker, computes the sum of
+    # two double-double numbers and return high double.  References:
+    # [1] http://gdz.sub.uni-goettingen.de/dms/load/img/?PPN=PPN362160546_0018&DMDID=DMDLOG_0023&LOGID=LOG_0023&PHYSID=PHYS_0232
```

Comment:
The two links provides two versions of the same paper.  The second one is a DOI, so should be permanent by definition of DOI, but the article is under a paywall there.  Regarding the first link, I don't speak German, but the site looks like a digital library.  http://www.digizeitschriften.de/dms/img/?PID=GDZPPN001170007 is a shorter URL and it's where the link on top of the page links to.